widget/embedded/src/store/quote.ts

```diff
@@ -29,6 +34,21 @@ export const getUsdValue = (
 export type Meta = Pick<MetaResponse, 'blockchains' | 'tokens'>;
 
 export type SetTokenParams = { token: Token; meta: Meta } | { token: null };
+export type SelectedQuote = {
+  swaps: SwapResult[];
+  requestId: string;
+  outputAmount?: string;
+  resultType?: RoutingResultType;
+  tags?: RouteTag[];
+  validationStatus: BlockchainValidationStatus[] | null;
+  requestAmount: string;
+};
+interface SomeQuoteState {
+  quotes: MultiRouteResponse | null;
+  selectedTag: RouteTag | null;
+  sortStrategy: PreferenceType;
+  refetchQuote: boolean;
+}
```

Comment:
Prefer using the type keyword over interface to avoid declaration merging.

Reply:
fixed